### DISCUSSION TITLE
Make stationary tools default when crafting

### DISCUSSION THREAD

**Discussion by Krwak** [cataclysmbn/Cataclysm-BN#2051] [General]
# Is your feature request related to a problem? Please describe.

Currently when crafting/repairing and there is a possibility to use a vehicle/base station for that, game defaults to closest tool to the character which often uses charges which have to be reloaded after use.
There is a window which appears where game asks which tool to use and the stationary ones are always on the end of this window.
![image](https://user-images.githubusercontent.com/72284934/135831440-ae642279-4498-4cdd-9843-77362a192b59.png)

# Describe the solution you'd like

I would like to have stationary tools being first when game gives the prompt about which tool to use.
It would probably require game to check for mounted vehicle tools, cause currently it goes by distance from PC.

# Describe alternatives you've considered

Make this a possibility in the options to be toggled on or off.

# Additional context

**Comment by ThinkInvis** [comment]
This should already be happening, as per: https://github.com/cataclysmbnteam/Cataclysm-BN/blob/8ef70c2e55373977fd8de9f45fc1666db7c96aa9/src/crafting.cpp#L1635-L1637
No clue why it isn't.
